# Mapping scRNAseq to 3D Spatial Coordinates

The below figure displays 5 histology itssue slides. Each row represents single cells mapped to a grid in the X, Y, Z planes (dorsal-ventral, rostral-caudal, superior-inferior). Cell abundance is represented by the intensity of a dot - indicating density of gene expression in that region.

The top 4 slides are of gene expression observed in Spatial Transcriptomic data. Only the first column of these 4 are observed and trained on, the extra axes are predicted cell positions. 

The bottom (5th and last) slide is predicted from scRNA-seq only of the dataset from the Allen Institute Data Challenge #2 (CAST) Critical Assessment through Spatially-resolved Transcriptomics. 

Based on analyses and results in the figures and plots, it is hypothesised that the data comes from the hippocampal/isocortex brain region from a publication of the Allen Institute:

A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation

Yao, Zizhen et al.

Cell, Volume 184, Issue 12, 3222 - 3241.e26



![](https://raw.githubusercontent.com/AskExplain/scRNA_maps/main/scMaps/mouse___Olfm3.png)



All information currently found at:

https://github.com/AskExplain/scRNA_maps

